### PR TITLE
Run phpstan checks.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Shell/Task/ModelTask.php [hunk 1]
```diff
@@ -1031,14 +1031,14 @@ protected function _getAllTables()
                 'Connections need to implement schemaCollection() to be used with bake.'
             );
 
-            return $this->_stop();
+            $this->_stop();
```
Comment (markstory) [inline src/Shell/Task/ModelTask.php]: You might need this return for when stop is mocked. You could also use abort() instead.

**Comment by ADmad** [comment]
@ondrejmirtes Is there a better way to deal with [this](https://travis-ci.org/cakephp/bake/jobs/310083150#L603) error other than just ignoring it?  The issue is caused by use of [@property](https://github.com/cakephp/bake/blob/de561b9674e74c30b6712b45408b90da16d6b838/src/View/Helper/DocBlockHelper.php#L76) tag within description text.

**Comment by ondrejmirtes** [comment]
@ADmad Your problem is not unique, I encountered it in at least three other places. We'll try to ignore phpDocs in the middle of text descriptions, but for now, try to construct as specific regex as possible to ignore it (so that you don't ignore other valid errors).

**Comment by ADmad** [comment]
This is good to go.

File: src/Shell/Task/ModelTask.php [hunk 2]
```diff
@@ -1027,18 +1027,14 @@ protected function _getAllTables()
     {
         $db = ConnectionManager::get($this->connection);
         if (!method_exists($db, 'schemaCollection')) {
-            $this->err(
+            $this->abort(
                 'Connections need to implement schemaCollection() to be used with bake.'
             );
-
-            return $this->_stop();
         }
```
Comment (saeideng) [inline src/Shell/Task/ModelTask.php]: > Connections **need to** implement schemaCollection() to be used with bake. 

and continue to use `$db->schemaCollection();`  :)

File: src/Shell/Task/ModelTask.php [hunk 3]
```diff
@@ -1027,18 +1027,14 @@ protected function _getAllTables()
...
         $schema = $db->schemaCollection();
         $tables = $schema->listTables();
         if (empty($tables)) {
-            $this->err('Your database does not have any tables.');
-
-            return $this->_stop();
+            $this->abort('Your database does not have any tables.');
```
Comment (saeideng) [inline src/Shell/Task/ModelTask.php]: and here ...

**Pull request merged**